trino-docs.sh

```diff
@@ -40,6 +40,7 @@ find -H "$VERSION" -type f -name '*.html' -print0 | xargs -0 perl -pi -e \
       window._linkedin_data_partner_ids = ['$LINKEDINID'];
     </script>
     <script async src=\"https://snap.licdn.com/li.lms-analytics/insight.min.js\"></script>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
```

Comment:
Apparently this is how we do it on trino.io: https://github.com/trinodb/trino.io/search?q=font+awesome